client/src/api/resources/projects.js

```diff
@@ -1,5 +1,7 @@
 export default {
-  index(filter = {}, page = {}) {
+  index(filterParams = {}, page = {}, includeDrafts = false) {
```

Comment:
This shouldn't be a client application concern at all, IMO. The API should decide what to return to the user — if the user is an admin, drafts are included in results. If they are not an admin, they can't see any drafts. If we handle it in the client rather than the server, we're not really hiding anything from the user.